src/components/services/DeepLinkingManager.tsx

```diff
@@ -74,15 +80,35 @@ export function DeepLinkingManager(props: Props) {
         }
       }
 
+      /** Handler for push messages received while app is in the foreground. */
+      const handleForegroundPushMessage = (message: FirebaseMessagingTypes.RemoteMessage) => {
+        if (message.notification?.body == null) {
+          console.warn('FirebaseMessagingTypes.RemoteMessage (foreground push message) has no body')
+          return
+        }
+
+        // Show a FlashNotification:
+        Airship.show(bridge => (
+          <FlashNotification
+            bridge={bridge}
+            message={message.notification?.body ?? ''}
```

Comment:
If we do `const body = message.notification?.body; if (body == null) console.warn(...`, then we don't need the `?? ''` part here, since we can just use `body` directly.

Reply:
Yeah, that's what I would think, but Typescript disagrees. Did another workaround